addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,96 @@
+# Name: xpaas
+# Description: Imports xPaaS templates
+# Url: https://github.com/jboss-openshift/application-templates/tree/ose-v1.3.6
+
+oc create -f jboss-image-streams.json -n openshift
```

Comment:
@hferentschik right now this is what a single image stream file we are using and we don't have any duplication for it. @jorgemoralespou suggested in case we need to split template but imagestream same then `apply` make more sense.